docs/sources/administration/roles-and-permissions/access-control/plan-rbac-rollout-strategy/index.md

```diff
@@ -170,63 +170,91 @@ curl --location --request POST '<grafana_url>/api/access-control/roles/' \
 
 By default, only a Grafana Server Admin can create and manage custom roles. If you want your `Editors` to do the same, [update the `Editor` basic role permissions]({{< ref "./manage-rbac-roles.md#update-basic-role-permissions" >}}). There are two ways to achieve this:
 
-- Add the `fixed:roles:writer` role permissions to the `basic:editor` role using the `role > from` list of your provisioning file:
-
-```yaml
-apiVersion: 2
-
-roles:
-  - name: 'basic:editor'
-    global: true
-    version: 3
-    from:
-      - name: 'basic:editor'
-        global: true
-      - name: 'fixed:roles:writer'
-        global: true
-```
-
-- Or add the following permissions to the `basic:editor` role, using provisioning or the [RBAC HTTP API]({{< relref "../../../../developers/http_api/access_control/#update-a-role" >}}):
-
-| action         | scope                       |
-| -------------- | --------------------------- |
-| `roles:read`   | `roles:*`                   |
-| `roles:write`  | `permissions:type:delegate` |
-| `roles:delete` | `permissions:type:delegate` |
+- Add the following permissions to the `basic:editor` role, using provisioning or the [RBAC HTTP API]({{< relref "../../../../developers/http_api/access_control/#update-a-role" >}}):
+
+  | action         | scope                       |
+  | -------------- | --------------------------- |
+  | `roles:read`   | `roles:*`                   |
+  | `roles:write`  | `permissions:type:delegate` |
+  | `roles:delete` | `permissions:type:delegate` |
+
+  As an example, here is a small bash script that fetches the role, modifies it using `jq` and updates it:
+
+  ```bash
+  # Fetch the role, modify it to add the desired permissions and increment its version
+  curl -H 'Authorization: Bearer glsa_kcVxDhZtu5ISOZIEt' \
+    -X GET '<grafana_url>/api/access-control/roles/basic_editor' | \
+    jq 'del(.created)| del(.updated) | del(.permissions[].created) | del(.permissions[].updated) | .version += 1' | \
+    jq '.permissions += [{"action": "roles:read", "scope": "roles:*"}, {"action": "roles:write", "scope": "permissions:type:delegate"}, {"action": "roles:delete", "scope": "permissions:type:delegate"}]' > /tmp/basic_editor.json
+
+  # Update the role
+  curl -H 'Authorization: Bearer glsa_kcVxDhZtu5ISOZIEt' -H 'Content-Type: application/json' \
+    -X PUT-d @/tmp/basic_editor.json '<grafana_url>/api/access-control/roles/basic_editor'
+  ```
+
+- Or add the `fixed:roles:writer` role permissions to the `basic:editor` role using the `role > from` list of your provisioning file:
```

Comment:
The key in the YAML snippet is `roles` not `role`.

We encourage that you use dot syntax to refer to nested fields: https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#refer-to-nested-field so it would be something like:

```suggestion
- You can instead add the `fixed:roles:writer` role permission to the `basic:editor` role by extending the `roles.from` list in your provisioning file:
```